crates/sc-proof-of-time/Cargo.toml

```diff
@@ -33,4 +33,5 @@ sp-runtime = { version = "24.0.0", git = "https://github.com/subspace/polkadot-s
 subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives" }
 subspace-proof-of-time = { version = "0.1.0", path = "../subspace-proof-of-time" }
 thread-priority = "0.16.0"
+tokio = "1.35.1"
```

Comment:
Please add `sync` feature or else compilation of the crate in isolation will fail, it only succeeds in a project due to feature unification.